pkg/collector/corechecks/ebpf/probe/ebpfcheck/probe.go

```diff
@@ -378,22 +378,42 @@ func arrayMemoryUsage(info *ebpf.MapInfo, nrCPUS uint64) (max uint64, rss uint64
 	return usage, usage
 }
 
-const sizeofHtabElem = 48
-const sizeOfBucket = 16
+const sizeofHtab = uint64(704)    // struct bpf_htab
```

Comment:
Is there no way to get this size with BTF or kernel headers?

Reply:
There is, but minified BTF would not include these types because we don't use them anywhere in eBPF. This is a good follow up task.